Mathlib/Combinatorics/SetFamily/Compression/UV.lean

```diff
@@ -75,100 +75,113 @@ section GeneralizedBooleanAlgebra
 variable [GeneralizedBooleanAlgebra α] [DecidableRel (@Disjoint α _ _)]
   [DecidableRel ((· ≤ ·) : α → α → Prop)] {s : Finset α} {u v a b : α}
 
-attribute [local instance] decidableEqOfDecidableLE
-
 /-- UV-compressing `a` means removing `v` from it and adding `u` if `a` and `u` are disjoint and
 `v ≤ a` (it replaces the `v` part of `a` by the `u` part). Else, UV-compressing `a` doesn't do
 anything. This is most useful when `u` and `v` are disjoint finsets of the same size. -/
 def compress (u v a : α) : α :=
   if Disjoint u a ∧ v ≤ a then (a ⊔ u) \ v else a
 #align uv.compress UV.compress
 
+theorem compress_of_disjoint_of_le (hua : Disjoint u a) (hva : v ≤ a) :
+    compress u v a = (a ⊔ u) \ v :=
+  if_pos ⟨hua, hva⟩
+#align uv.compress_of_disjoint_of_le UV.compress_of_disjoint_of_le
+
+theorem compress_of_disjoint_of_le' (hva : Disjoint v a) (hua : u ≤ a) :
+    compress u v ((a ⊔ v) \ u) = a := by
```

Comment:
I don't think I would guess this lemma name from the LHS; is `compess_sup_sdiff_of_disjoint_of_le` sensible?

Reply:
Oh, I see this is just a move.

Reply:
Yeah sorry it's not a great name, but so far it hasn't been a problem.